README.md

```diff
@@ -83,12 +83,15 @@ await runner.setup(__dirname);
 
 Optionally, you can provide "setup" files if you want to copy additional files into the target directory, say from _node_modules_ or a fixtures folder.  You can provide these files as an array of objects.
 
-* `source`: path of the file to copy
-* `destination`: path of where to copy the file to
+- `source`: path of the file to copy (can be a string or URL object)
```

Comment:
we can still keep the same character for the bullet (the `*`).  Let's try and keep changes to a minimum where possible 👍